### PR TITLE
Client Registration Doc updates

### PR DESCRIPTION
**Pull request by tthompson-okta**
-Added PostMan collection for client registration API.
-Updated client registration API doc to include "application_type" attribute.
-Fixed a few bad examples in the Social Authentication section.

Resolves: OKTA-81488
Resolves: OKTA-81398
Resolves: OKTA-81319

@joelfranusic-okta @mystiberry-okta @hassenkaraa-okta @karlmcguinness-okta @federations-okta

### CODE REVIEW

File: _source/docs/api/resources/oauth-clients.md [hunk 1]
```diff
@@ -61,6 +62,7 @@ Client applications have the following properties:
 | client_name                | human-readable string name of the client application              | String                                                                 | FALSE    | FALSE  | FALSE    |
 | client_secret              | OAuth 2.0 client secret string (used for confidential clients)    | String                                                                 | TRUE     | TRUE   | TRUE     |
 | logo_uri                   | URL string that references a logo for the client                  | String                                                                 | TRUE     | FALSE  | FALSE    | 
+| application_type           | The type of client application                                    | `web`, `native`, or `browser`                                          | TRUE     | TRUE   | TRUE     |
```
Comment (yuliu-okta) [inline _source/docs/api/resources/oauth-clients.md]: Could you please also add initiate_login_uri? Then I do not need to submit another PR:)

Comment (tthompson-okta) [inline _source/docs/api/resources/oauth-clients.md]: Yep will do.

Comment (tthompson-okta) [inline _source/docs/api/resources/oauth-clients.md]: Done.

Comment (mystiberry-okta) [inline _source/docs/api/resources/oauth-clients.md]: Looks good.

File: _source/docs/api/resources/oauth-clients.md [hunk 2]
```diff
@@ -61,10 +63,12 @@ Client applications have the following properties:
...
 | redirect_uris              | array of redirection URI strings for use in redirect-based flows  | Array                                                                  | TRUE     | FALSE  | FALSE    |
```
Comment (yuliu-okta) [inline _source/docs/api/resources/oauth-clients.md]: it should mention that there is only one uri allowed.

File: _source/docs/api/resources/oauth-clients.md [hunk 3]
```diff
@@ -61,10 +63,12 @@ Client applications have the following properties:
...
 | response_types             | array of OAuth 2.0 response type strings                          | Array of `code`, `token`, `id_token`                                   | TRUE     | FALSE  | FALSE    |
 | grant_types                | array of OAuth 2.0 grant type strings                             | Array of `authorization_code`, `implicit`, `password`, `refresh_token` | FALSE    | FALSE  | FALSE    | 
 | token_endpoint_auth_method | requested authentication method for the token endpoint            | `none`, `client_secret_post`, or `client_secret_basic`                 | FALSE    | FALSE  | FALSE    |
+| initiate_login_uri         | URL that a third party can use to initiate a login by the client  | String                                                                 | TRUE     | FALSE  | FALSE    |
```
Comment (yuliu-okta) [inline _source/docs/api/resources/oauth-clients.md]: Should we mention that if initiate_login_uri is missing, Okta will try to initiate an IdP login by post id_token to the redirect uri?

**Comment by mystiberry-okta** [comment]
@tthompson-okta okay to publish now? Or do I need to wait for Preview or Prod? Looks like now is good, just double-checking cuz I'm still a newbie.

File: _source/docs/api/resources/oauth-clients.md (hunk 2 above)
Comment (tthompson-okta) [inline _source/docs/api/resources/oauth-clients.md]: Hang on, that's not necessarily true. This API should always allow multiple redirect_uris on a create/POST call. The only time it would restrict to 1 is if the admin had gone in (via the UI) and turned on the Okta-style Idp-init flow. After that if you were to do a PUT/update of this API, it should return an error indicating a constraint violation because of Idp-init settings.

**Comment by tthompson-okta** [comment]
Let me make some clarifications per Yu's comments, and then I'll give you the go ahead.

File: _source/docs/api/resources/oauth-clients.md (hunk 2 above)
Comment (tthompson-okta) [inline _source/docs/api/resources/oauth-clients.md]: Per our latest discussion, we will continue to always support multiple redirect URIs, so I think this is fine as-is.

File: _source/docs/api/resources/oauth-clients.md (hunk 3 above)
Comment (tthompson-okta) [inline _source/docs/api/resources/oauth-clients.md]: It might be nice to have a discussion of the different Idp-init styles in the OIDC documentation (oidc.md), but since this page is specific to the spec-compliant API endpoint, I think we should just rely on the spec to inform people of what this is meant for.

**Comment by tthompson-okta** [comment]
Okay @mystiberry-okta I think this is fine as-is. More tweaks and updates will be added in future PRs.
